routers/routes/routes.go

```diff
@@ -613,6 +613,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 			m.Get("/*", repo.WikiRaw)
 		}, repo.MustEnableWiki)
 
+		m.Group("/activity", func() {
+			m.Get("", repo.Activity)
+			m.Get("/:period", repo.Activity)
+		}, context.RepoRef(), repo.MustBeNotBare, context.CheckUnit(models.UnitTypeCode))
```

Comment:
This is a sub feature of unit `Code`?

Reply:
I was thinking at first it should be code but now I don't know actually... What do you think?

Reply:
I think it should be available to users who have permission to view PR and Issues.

Reply:
@lafriks I agree with @daviian, if you have `Issues` or `Pull Request` you could visit the activity.